[2022-07-27 22:23:38,781] INFO 130 configuration.py get_training_pipeline_config() Training pipleine config: TrainingPipelineConfig(artifact_dir='D:\\Storessalesproject\\storessales\\artifact')
[2022-07-27 22:23:38,781] INFO 68 configuration.py get_data_ingestion_config() Data Ingestion config: DataIngestionConfig(dataset_download_url='https://github.com/krishnavizster/storedatatgz/blob/main/storessalesdata.tgz', tgz_download_dir='D:\\Storessalesproject\\storessales\\artifact\\data_ingestion\\2022-07-27-22-23-37\\tgz_data', raw_data_dir='D:\\Storessalesproject\\storessales\\artifact\\data_ingestion\\2022-07-27-22-23-37\\raw_data', ingested_train_dir='D:\\Storessalesproject\\storessales\\artifact\\data_ingestion\\2022-07-27-22-23-37\\ingested_data\\train', ingested_test_dir='D:\\Storessalesproject\\storessales\\artifact\\data_ingestion\\2022-07-27-22-23-37\\ingested_data\\test')
[2022-07-27 22:23:38,781] INFO 21 data_injection.py __init__() >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>Data Ingestion log started.<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<< 
[2022-07-27 22:23:38,782] INFO 43 data_injection.py download_storessales_data() Downloading file from :[https://github.com/krishnavizster/storedatatgz/blob/main/storessalesdata.tgz] into :[D:\Storessalesproject\storessales\artifact\data_ingestion\2022-07-27-22-23-37\tgz_data\storessalesdata.tgz]
[2022-07-27 22:23:39,501] INFO 45 data_injection.py download_storessales_data() File :[D:\Storessalesproject\storessales\artifact\data_ingestion\2022-07-27-22-23-37\tgz_data\storessalesdata.tgz] has been downloaded successfully.
[2022-07-27 22:23:39,501] INFO 60 data_injection.py extract_tgz_file() Extracting tgz file: [D:\Storessalesproject\storessales\artifact\data_ingestion\2022-07-27-22-23-37\tgz_data\storessalesdata.tgz] into dir: [D:\Storessalesproject\storessales\artifact\data_ingestion\2022-07-27-22-23-37\raw_data]
[2022-07-27 22:23:39,503] ERROR 16 demo.py main() 
        Error occured in script: 
        [ D:\Storessalesproject\storessales\pipeline\pipeline.py ] at 
        try block line number: [68] and exception block line number: [70] 
        error message: [
        Error occured in script: 
        [ D:\Storessalesproject\storessales\pipeline\pipeline.py ] at 
        try block line number: [45] and exception block line number: [48] 
        error message: [
        Error occured in script: 
        [ D:\Storessalesproject\storessales\component\data_injection.py ] at 
        try block line number: [128] and exception block line number: [131] 
        error message: [
        Error occured in script: 
        [ D:\Storessalesproject\storessales\component\data_injection.py ] at 
        try block line number: [61] and exception block line number: [66] 
        error message: [file could not be opened successfully]
        ]
        ]
        ]
        
[2022-07-27 22:23:39,506] INFO 136 data_injection.py __del__() >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>Data Ingestion log completed.<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<< 


